Jenkinsfile

```diff
@@ -99,5 +109,45 @@ pipeline {
         """
       junit testResults: "logs/junit.xml", keepLongStdio: false
     }
+    success {
+      script {
+        if( env.UMB_MESSAGE in [true, 'true'] ) {
+          def operator_image = "${env.OCS_OPERATOR_IMAGE}"
+          def operator_version = operator_image.split(':')[-1]
+          def properties = """
+            TOOL=ocs-ci
+            PRODUCT=ocs
+            PRODUCT_VERSION={operator_version}
+            PRODUCT_BUILD_CAUSE=${BUILD_CAUSE}
+            OCS_OPERATOR_DEPLOYMENT=${env.OCS_OPERATOR_DEPLOYMENT}
+          """
+          def contentObj = [
+            "SENDER_BUILD_NUMBER": "${BUILD_NUMBER}",
+            "OCS_OPERATOR_IMAGE": "${env.OCS_OPERATOR_IMAGE}",
```

Comment:
Ah, I think we will need to have also the REGISTRY IMAGE which is actually the most important for us as this is one which will be used. Can you add this as well please?  I mean this one https://quay.io/repository/rhceph-dev/ocs-registry?tab=tags , as this is where the catalog source will be pointing for CSV. Thanks

Reply:
There's no mention of `registry_image` in this repo, that I can see. Is it really worth holding up this PR longer?

Reply:
Based on your comment here: https://github.com/red-hat-storage/ocs-ci/pull/713#discussion_r323905839 I renamed `ocs_operator_cs_image` to just `ocs_operator_image` but didn't realized at that moment that it's actually registry image for CS (catalog source) and not ocs_operator image where the name with cs abbreviation of catalog source were more correct. 

So to be correct I will submit another PR to change this parameter to `ocs_operator_registry_image` or back to cs but registry fits more as it's name of your registry.

So what we need to set for deployment is quay.io/rhceph-dev/ocs-registry and not ocs-operator image as ocs-operator image is set in CSV in the registry.

Reply:
Just to make sure I'm clear:
1. Are we dropping `ocs_operator_image` then?
2. `ocs_operator_registry_image` or just `ocs_registry_image`?

Reply:
FWIW, this PR passes both parameters.

Reply:
1) you can keep both, we can use this info for some report or email or whatever.
2) we can use shorter version